app/services/replication_transfer_updater.rb

```diff
@@ -83,7 +83,7 @@ def request_storage
       if params[:fixity_value] == correct_fixity
         record.update(store_requested: true)
       else
-        record.cancel!("fixity_reject")
+        record.cancel!("fixity_reject", params[:cancel_reason_detail])
       end
```

Comment:
Hmm, I don't know what information this updater would want to put into the cancel_reason_detail.  The reported and expected fixities are already available.  With future nonce support, we could report what we expected, or keep that information hidden.  Thoughts?